integrationtest/batch-jsr352/src/test/java/org/hibernate/search/integrationtest/batch/jsr352/massindexing/MassIndexingJobWithMultiTenancyIT.java

```diff
@@ -70,6 +72,10 @@
 
 	@Before
 	public void setUp() throws Exception {
+		assumeTrue( "Run this test only when the default persistence unit is in use",
+				PersistenceUnitTestUtil.isDefaultPersistenceUnit()
+		);
```

Comment:
Alternatively, run the test with Elasticsearch too?

You could rely on the pre-existing `org.hibernate.search.util.impl.integrationtest.common.rule.BackendConfiguration#BACKEND_TYPE` util instead of the custom `org.hibernate.search.jsr352.persistence_unit` property:

* In the failsafe config, set `org.hibernate.search.integrationtest.backend.type` to either `lucene` or `elasticsearch`
* In `PersistenceUnitTestUtil`, infer the PU name from `org.hibernate.search.util.impl.integrationtest.common.rule.BackendConfiguration#BACKEND_TYPE`.
* In `MassIndexingJobWithMultiTenancyIT`, infer the `BackendConfiguration` (either `LuceneBackendConfiguration` or `ElasticsearchBackendConfiguration`) from `org.hibernate.search.util.impl.integrationtest.common.rule.BackendConfiguration#BACKEND_TYPE`. Alternatively, move the if else to a dedicated `BackendConfigurations` class, like we do in the documentation; see `org.hibernate.search.documentation.testsupport.BackendConfigurations`.